src/components/Navbar/Navbar.scss

```diff
@@ -1,32 +1,36 @@
 .Navbar {
-  width: 100%;
+  width: 90%;
   height: 50px;
```

Comment:
50px height might be to small, per design it is about 100px
However, not really important now, I guess